src/pages/errorpages/HandleNotFound.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
```

Comment:
Sikkert en egen task, men vi burde vel også vist 404 for ressurser som ikke finnes: Ex: http://localhost:3000/resources/fdsfsd  Nå får vi feilmelding og 401

Reply:
Hvordan skiller vi mellom ressurser som ikke finnes og ressurser som skulle gitt 404? Nå får vi 401 uansett om vi har tilgang til ressursen eller om den ikke eksisterer?

Reply:
Om vi ikke får mer fornuftig fra backend - kan vi kanskje bare lage en mer generisk tekst istedenfor feilmeldng  ("ressursen finnes ikke eller du har ikke tilgang" elns).

Reply:
Lager egen task på det.